modyn/config/schema/pipeline/evaluation/strategy/periodic.py

```diff
@@ -0,0 +1,28 @@
+from functools import cached_property
+from typing import Literal
+
+from modyn.const.regex import REGEX_TIME_UNIT
+from modyn.utils.utils import SECONDS_PER_UNIT
+from pydantic import Field
+
+from ._interval_strategy import _IntervalEvalStrategyConfig
+
+
+class PeriodicEvalStrategyConfig(_IntervalEvalStrategyConfig):
+    """Comparable to TimeTriggerConfig and DataAmountTriggerConfig, but for scheduling evaluation runs."""
```

Comment:
I don't get that docstring yet. Scheduling runs sounds to me like this defines when the evaluation is executed. Maybe we should update this docstring to describe how the intervals are generated